lib/commands/context.js

```diff
@@ -108,7 +106,8 @@ commands.mobileGetContexts = async function mobileGetContexts () {
     androidDeviceSocket: this.opts.androidDeviceSocket,
     ensureWebviewsHavePages: true,
     webviewDevtoolsPort: this.opts.webviewDevtoolsPort,
-    enableWebviewDetailsCollection: true
+    enableWebviewDetailsCollection: true,
+    isChromeSession: this.isChromeSession
```

Comment:
This parameter changes the response when `this.isChromeSession` is true. (though I assume it's ok)
Let me know if we should keep the original behaviour.